src/WebJobs.Script/Scale/ScaleStatusResult.cs

```diff
@@ -15,6 +15,6 @@ namespace Microsoft.Azure.WebJobs.Script.Scale
     /// </remarks>
     public class ScaleStatusResult
     {
-        public ScaleVote Vote { get; set; }
+        public int Vote { get; set; }
```

Comment:
I don't think we should change this public protocol contract. We can still set the enum value correctly based on current worker count and the new target worker count in the case of TBS

Reply:
This is also intersected with @mathewc comments:
https://github.com/Azure/azure-webjobs-sdk/pull/2918#discussion_r980318906
https://github.com/Azure/azure-webjobs-sdk/pull/2918#discussion_r980319382

I am calculating the worker count difference in an extension. If we are introducing the new property the SC will be responsible to determinate the difference in workers.

Reply:
We can mark this as resolved, discussed offline and we should be keeping the current contract for ScaleVote, and add a separate attribute `TargetWorkerCount`